.gitignore

```diff
@@ -1,3 +1,6 @@
 target
 /.idea
 /*.iml
+companies.csv
```

Comment:
umjesto konkretnih `.csv` datoteka, u .gitignore stavi da opcenito ignorira CSV datoteke, to ce ti biti nesto ovakvo, ali provjeri za svaki slucaj: `*.csv`

Reply:
Može